src/fdonotification.c

```diff
@@ -414,6 +430,7 @@ fdo_add_notification (GDBusConnection *connection,
       n->id = g_strdup (id);
       n->notify_id = 0;
       n->activate_action = activate_action;
+      n->activation_token = NULL;
```

Comment:
This and notify_id are not required to be set. The allocation gives us 0/NULL.

Reply:
let's still keep it to stay consistent with the code style.